qt/scientific_interfaces/Muon/ALCDataLoadingPresenter.cpp

```diff
@@ -419,7 +421,8 @@ bool ALCDataLoadingPresenter::checkCustomGrouping() {
  * @returns :: True if grouping OK, false if bad
  */
 std::string ALCDataLoadingPresenter::isCustomGroupingValid(const std::string &group, bool &isValid) {
-  if (!std::isdigit(group[0]) || std::any_of(std::begin(group), std::end(group), ::isalpha)) {
```

Comment:
Forgive me as I'm likely not seeing it but would
```cpp
if(!std::any_of(std::begin(group), std::end(group), ::isdigit) {
  isValid = false;
  return "";
}
```
not be what is required?

Reply:
it can have commas in the list

Reply:
Thanks.